frontend/public/components/modals/delete-pvc-modal.tsx

```diff
@@ -72,6 +79,8 @@ const DeletePVCModal = withHandlePromise<DeletePVCModalProps>((props) => {
 
 export type DeletePVCModalProps = {
   pvc: PersistentVolumeClaimKind;
+  redirectTo?: string;
```

Comment:
It looks like this prop is no longer used and can be removed from the types?

Reply:
> It looks like this prop is no longer used and can be removed from the types?

@spadgett Updated.